sgkit_bgen/bgen_reader.py

```diff
@@ -107,15 +107,27 @@ def __getitem__(self, idx):
             raise IndexError(  # pragma: no cover
                 f"Indexer must be two-item tuple (received {len(idx)} slices)"
             )
+        if not all(isinstance(i, slice) or isinstance(i, int) for i in idx):
+            raise IndexError(  # pragma: no cover
```

Comment:
Is it possible to test this case?

Reply:
https://github.com/pystatgen/sgkit-bgen/pull/15/commits/74339918906a0f7af5daee1a3b10b7c52ab7a450